lib/cluster.js

```diff
@@ -341,9 +341,9 @@ function masterInit() {
       process: workerProcess
     });
 
-    worker.on('message', (message, handle) =>
-      cluster.emit('message', message, handle)
-    );
+    worker.on('message', function(message, handle) {
+      cluster.emit('message', this, message, handle);
```

Comment:
I can change it if you think it's clearer.  The way it's written now saves a few bytes of memory and reduces GC strain infinitesimally (one fewer closure context variable to traverse.)